lib/build/config.js

```diff
@@ -14,8 +14,7 @@ function getCapacitorMessage() {
         return `create-supertokens-app uses ${chalk.bold("nextjs-native-starter")} as a template app for Capacitor. ${chalk.bold("nextjs-native-starter")} is a starter project created by ${chalk.blue("https://github.com/RobSchilderr")}, if you face any problems please open an issue here: ${chalk.blue("https://github.com/RobSchilderr/nextjs-native-starter/issues")} or reach out to us on the SuperTokens Discord server: ${chalk.blue("https://supertokens.com/discord")}.\n\nTo get started refer to the README on Github ${chalk.blue("https://github.com/RobSchilderr/nextjs-native-starter")}`;
     }
 }
```

Comment:
one question is that if a user is using `pnpm` for example, the generated demo apps (for node + react) will stilljust use npm. Do you think that's ok?

Reply:
Do you mean that the scripts in the `package.json` file will be run with `npm`, like `npm run start`? Sure, that's completely okay!

These package managers do not have differences, when running these scripts. They vary in syntax (which is correctly handled here, I checked), but that's all.

Reply:
Gotcha!